src/app/screens/confirmNftTransaction/index.tsx

```diff
@@ -150,8 +152,27 @@ function ConfirmNftTransaction() {
 
   const handleOnConfirmClick = (txs: StacksTransaction[]) => {
     if (isLedgerAccount(selectedAccount)) {
-      const txType: LedgerTransactionType = 'STX';
-      navigate('/confirm-ledger-tx', { state: { unsignedTx, type: txType } });
+      const type: LedgerTransactionType = 'STX';
+      navigate('/confirm-ledger-tx', {
+        state: {
+          unsignedTx: txs[0].serialize(),
+          type,
+          recipients: [
```

Comment:
Same comment as https://github.com/secretkeylabs/xverse-web-extension/pull/564#discussion_r1307229349 . type Recipient prop should not be used for this